lightning/src/ln/channelmanager.rs

```diff
@@ -6180,6 +6180,165 @@ where
 								}
 								None
 							},
+							HTLCForwardInfo::AddHTLC(PendingAddHTLCInfo {
```

Comment:
I think commit structure is fine like this. As a reviewer also wouldn't mind squashing with the previous commit.